api/src/main/java/org/openmrs/api/impl/ConditionServiceImpl.java

```diff
@@ -117,37 +77,97 @@ public Condition getCondition(Integer conditionId) {
 	}
 
 	/**
-     * @see ConditionService#getAllConditions(Patient)
+	 * @see ConditionService#getAllConditions(Patient)
 	 */
 	@Override
 	@Transactional(readOnly = true)
 	public List<Condition> getAllConditions(Patient patient) {
 		return conditionDAO.getAllConditions(patient);
 	}
 
+	/**
+	 * @see ConditionService#getConditionsByEncounter(Encounter)
+	 */
+	@Override
+	public List<Condition> getConditionsByEncounter(Encounter encounter) throws APIException {
```

Comment:
This can also be `@Transactional(readOnly = true)`

Reply:
Could easily be, but I actually just moved the method rather than do anything to edit it. I'd prefer to stick that change as part of a different PR.

Reply:
So this is just a move?  (Cool, fine with me)